guides/common/assembly_configuring-puppet.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context-of-assembly_configuring-puppet: {context}]
+
+ifndef::context[]
+[id="assembly_configuring-puppet"]
+endif::[]
+ifdef::context[]
+[id="assembly_configuring-puppet_{context}"]
+endif::[]
+= Configuring Puppet
+
+:context: assembly_configuring-puppet
+
+[role="_abstract"]
+Puppet is an optional component in {ProjectX}.
+If you installed {Project} afresh and you want to continue using Puppet, you have to enable it in your {Project}.
+If you upgraded your {Project} and you want to stop using Puppet, you can disable it and purge Puppet data and content, respectively.
+
+include::modules/proc_enabling-puppet.adoc[leveloffset=+1]
+
+ifdef::satellite[]
+[role="_additional-resources"]
```

Comment:
```suggestion
```

Reply:
This is a template-given standard that we use for metadata in downstream publishing, therefore it will stay.
See https://github.com/theforeman/foreman-documentation/pull/890#discussion_r760410889